core/plannernotes.c

```diff
@@ -286,7 +329,13 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 						len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; color: red; float: left;'><b>%s %s</b></td>", gasname(&gasmix),
 									temp);
 					} else {
-							len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; color: red; float: left;'><b>%s</b></td>", gasname(&gasmix));
+						len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; color: red; float: left;'><b>%s</b></td>", gasname(&gasmix));
```

Comment:
Did you recognize that here it is `gasmix` what is used and not `newgasmix`?
You please have to double check if the lines below are really correct or if in the following lines `newgasmix` has to be replaced by `gasmix`. Plus will it then still work that the calculations are only done once?